sphinx/tutorial/cylc/runtime/runtime-configuration.rst

```diff
@@ -0,0 +1,456 @@
+
+      * Camborne - ``3808``
+      * Shetland - ``3005``
+      * Belmullet - ``3976``
+
+      Add three more ``get_observations`` tasks for each of the remaining
+      weather stations.
+
+      .. spoiler:: Solution warning
+
+         .. code-block:: cylc
+
+            [runtime]
+                [[get_observations_heathrow]]
+                    script = get-observations
+                    [[[environment]]]
+                        SITE_ID = 3772
+                        API_KEY = d6bfeab3-3489-4990-a604-44acac4d2dfb
+                [[get_observations_camborne]]
+                    script = get-observations
+                    [[[environment]]]
+                        SITE_ID = 3808
+                        API_KEY = d6bfeab3-3489-4990-a604-44acac4d2dfb
+                [[get_observations_shetland]]
+                    script = get-observations
+                    [[[environment]]]
+                        SITE_ID = 3005
+                        API_KEY = d6bfeab3-3489-4990-a604-44acac4d2dfb
+                [[get_observations_belmullet]]
+                    script = get-observations
+                    [[[environment]]]
+                        SITE_ID = 3976
+                        API_KEY = d6bfeab3-3489-4990-a604-44acac4d2dfb
+
+      Check the ``suite.rc`` file is valid by running the command:
+
+      .. code-block:: bash
+
+         cylc validate .
+
+      .. TODO: Add advice on what to do if the command fails.
+
+   #. **Test The** ``get_observations`` **Tasks.**
+
+      Next we will test the ``get_observations`` tasks.
+
+      Open the cylc GUI by running the following command:
+
+      .. code-block:: bash
+
+         gcylc runtime-tutorial &
+
+      Run the suite either by pressing the play button in the cylc GUI or by
+      running the command:
+
+      .. code-block:: bash
+
+         cylc run runtime-tutorial
+
+      If all goes well the suite will startup, the tasks will run and succeed.
+      Note that the tasks which do not have a ``[runtime]`` section will still
+      run though they don't do anything as they don't call any scripts.
+
+      Once the suite has reached the final cycle point and all tasks have
+      succeeded the suite will automatically shutdown.
+
+      .. TODO: Advise on what to do if all does not go well.
+
+      The ``get-observations`` script produces a file called ``wind.csv`` which
+      contains the wind-speed and direction. This file is written in the task's
+      :term:`work directory`.
+
+      Try and open one of the ``wind.csv`` files. Note the path to the
+      :term:`work directory` is:
+
+      .. TODO: Move/Remove?
+
+      .. code-block:: sub
+
+         work/<cycle-point>/<task-name>
+
+      You should find a file with four numbers:
+
+      * The longitude of the weather station.
+      * The latitude of the weather station.
+      * The wind direction *[direction the wind is blowing towards]* (in degrees).
```

Comment:
Wind direction is normally given from the direction it is blowing from (i.e. 270/Westerly/from the west).. don't know if this affects the tutorial suite calculations...(or if it even matters here)

Reply:
This is purposeful (the wind is flipped in the `get-observations` script.